zerver/lib/actions.py

```diff
@@ -2699,7 +2710,7 @@ def _default_stream_permision_check(user_profile: UserProfile, stream: Optional[
             raise JsonableError(_('Insufficient permission'))
 
 def do_change_default_sending_stream(user_profile: UserProfile, stream: Optional[Stream],
-                                     log: bool=True) -> None:
+                                     log: Union[bool, str]=True) -> None:
     _default_stream_permision_check(user_profile, stream)
```

Comment:
This change doesn't seem to have a foundation in this commit.

Reply:
@roberthoenig I'm not sure why that change was part of the PR, but it's removed now. Thanks!